### ISSUE TITLE
Can you please change the look of when you right click on the website?

### ISSUE THREAD

**Issue by ask-arim** [Deri-Kurniawan/Deri-Kurniawan.github.io#20]
Please make the look of a right-click look the same as the actual Windows 11 on [the website](https://deri-kurniawan.vercel.app). (BTW, I found out that it's possible by right-clicking anywhere on [this page](https://win11.vercel.app/). And you can ask @PiyushSuthar (the creator of that website) how to make [the website](https://deri-kurniawan.vercel.app/) have the same feature once he finishes doing it on [his version of this repository](https://github.com/PiyushSuthar/Windows-11-Web))
![Actual Windows 11](https://user-images.githubusercontent.com/111149452/203636546-e7289fd9-5d2d-4749-bc53-305d60fc4b29.png)

**Comment by PiyushSuthar** [comment]
I'll be straight, that's not how open source works.
Please stop mentioning me everywhere.

Thank you

**Comment by Deri-Kurniawan** [comment]
Yes, only mention if they have responsibility for, just mention repo if you need share something
